### DISCUSSION TITLE
Message customization

### DISCUSSION THREAD

**Discussion by Tohruchama** [jagrosh/MusicBot#1233] [Ideas]
Really loving the bot, having tried to set up a bot using code the past day or so makes me really appreciate youe work, guys.
Is there a chance of getting a feature to just like editing the commands, edit how the bot delivers it's messages. For example, editing the icons and words the bot sends whenever someone uses the "play" command or maybe saying "bye bye!" whenever someone uses the "shutdown" command?
I set up a bot with certain characteristics and it would add a lot of personality to it if I could change these settings.

**Comment by MichailiK** [comment]
We're more or less tracking this under #689. You could specify your own localization file & edit the messages there.

**Reply by Tohruchama** [reply]
Yeah, that's it! Thanks and sorry, it's a bit different from what I meant but would certainly do the trick.

**Reply by jagrosh** [reply]
Also, editing the emojis that the bot uses is already possible, it's in the config file.
